integrations/nifi/nifi-nar-bundles/nifi-core-bundle/nifi-core-processors/src/main/java/com/thinkbiganalytics/nifi/v2/ingest/RegisterFeedTables.java

```diff
@@ -109,6 +110,39 @@
         .allowableValues(TableType.FEED.toString(), TableType.VALID.toString(), TableType.INVALID.toString(), TableType.PROFILE.toString(), TableType.MASTER.toString(), ALL_TABLES)
         .defaultValue(ALL_TABLES)
         .build();
+    /**
+     * Property indicating which file system is to back hive tables
+     */
+    public static final PropertyDescriptor STORAGE_FORMAT = new PropertyDescriptor.Builder()
+        .name("Storage Type")
+        .description("Specifies if the hive tables are to be backed by HDFS or S3.")
+        .required(true)
+        .allowableValues(StorageType.S3.toString(), StorageType.HDFS.toString())
+        .defaultValue(StorageType.HDFS.toString())
+        .build();
+
+    private static String S3N = "s3n";
+    private static String S3A = "s3a";
+    /**
+     * Property indicating which protocol is to be used if using s3
+     */
+    public static final PropertyDescriptor S3_PROTOCOL = new PropertyDescriptor.Builder()
+            .name("S3 Protocol")
+            .description("Specifies which S3 protocol is to be used if using S3 storage.")
+            .required(false)
+            .allowableValues(S3N, S3A)
```

Comment:
I'm wondering if S3N and S3A are the only and correct choices here?   I read a little on S3 support in Hadoop 2.7 (https://wiki.apache.org/hadoop/AmazonS3) and it strongly encourages s3a for Hadoop2.7+. Can Kylo try to assert 2.7+ going forward?   It also states S3 will not be supported in Hadoop3.0 so it seems correct that is not a choice here.

According to this (http://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-plan-file-systems.html) S3N is strongly discouraged.

I'm wondering if s3n should be removed..